code/modules/virus2/effect/stage_4.dm

```diff
@@ -1148,6 +1159,32 @@
 		H.set_species("Unathi")
 		H.regenerate_icons()
 
+/datum/disease2/effect/insectoid
+	name = "Kafkaesque Syndrome"
+	desc =  "A previously experimental syndrome that found its way into the wild. Causes the infected to mutate into an Insectoid."
+	stage = 4
+	badness = EFFECT_DANGER_DEADLY
+
+/datum/disease2/effect/insectoid/activate(var/mob/living/mob)
```

Comment:
bad arg name, this may seem pedantic but please pick anything else

user or L would be popular in a case like this